### DISCUSSION TITLE
Crazyflie client logging data: settings->open config folder doesn't do anything

### DISCUSSION THREAD

**Discussion by ugrm** [bitcraze/discussions#1478] [Q&A]
I'm using the crazyflie client to log data (estimates, locsrv values, etc). I check both the start and the write to file option in the log blocks page inside the client. After the flight, clicking on the open config folder option does nothing. I tried looking through some folders but couldn't find it. Could anyone tell me where the logged data is saved? Thanks.

**Comment by knmcguire** [comment]
Hi! 

Could you let us know which OS you are using? 

The [cfclients](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/)'s debug tab (found under 'Help'>'about') can provide  information about your development environment and your connected Crazyflie. Also if you see any errors from the terminal you are running the cfclient from can help a lot
